composites/SnippetPreview/components/ProgressBar.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import colors from "../../../style-guide/colors.json";
+
+const ProgressBarContainer = styled.progress`
+	box-sizing: border-box;
+	width: 100%;
+	height: 8px;
+	display: block;
+	margin-top: 5px;
+	border: none;
+	-moz-appearance: none;
+	-webkit-appearance: none;
+	appearance: none;
+
+	::-webkit-progress-bar {
+	   	background-color: ${colors.$color_background_light};
+	   	color: ${( { progressColor } ) => progressColor };
```

Comment:
there are spaces on line 18 and 19, please use tabs.
Also, not sure about the syntax `${( { progressColor } ) => progressColor }` and why below a different syntax is used: `${ props => props.progressColor }`. I'd use always the second one, also in the other cases.